frontend/src/features/properties/map/research/common/ResearchHeader.tsx

```diff
@@ -26,7 +27,11 @@ const ResearchHeader: React.FunctionComponent<IResearchHeaderProps> = props => {
           </Col>
           <Col className="text-right">
             Created: <strong>{prettyFormatDate(researchFile?.appCreateTimestamp)}</strong> by
-            <span> {researchFile?.appCreateUserid} </span>
+            <UserNameTooltip
+              key={'appCreateBy'}
```

Comment:
if this is a string you can just use "" instead of {''}

Reply:
are you using the key within the usernametooltip?

Reply:
For now I have removed the `key` as I thought its not needed for now.